src/Game.tsx

```diff
@@ -31,7 +32,12 @@ export default function Game() {
 
 	const joinGame = () => {
 		if (playerId) {
-			sendJsonMessage({ type: 'JOIN_GAME', gameId, playerId, name: playerName });
+			sendJsonMessage({ id: v4(), type: 'JOIN_GAME', gameId, playerId, name: playerName });
+		}
+	}
+	const createGame = () => {
+		if (playerId) {
+			sendJsonMessage({ id: v4(), type: 'CREATE_GAME', gameId, name: playerName, avatarSetId: '46efff1b-5ca2-57fc-8e98-f1bad529f45f' });
```

Comment:
avatarSetId should not be hardcoded, instead it should be one of the avatar sets determined by the call to get all avatar sets